rho/utilities.py

```diff
@@ -18,16 +18,24 @@
 import tempfile
 from shutil import move
 import sh
+from xdg.BaseDirectory import xdg_data_home, xdg_config_home
 from rho.translation import _
 
 if sys.version_info > (3,):
     import _thread
 else:
     import thread as _thread  # pylint: disable=import-error
 
-
-CREDENTIALS_PATH = 'data/credentials'
-PROFILES_PATH = 'data/profiles'
+RHO_PATH = 'rho'
+CONFIG_DIR = os.path.join(xdg_config_home, RHO_PATH)
+DATA_DIR = os.path.join(xdg_data_home, RHO_PATH)
+RHO_LOG = os.path.join(CONFIG_DIR, 'rho_log')
```

Comment:
I think `RHO_LOG` makes more sense in `DATA_DIR`, not `CONFIG_DIR`

Reply:
I'll update this. Thanks for the feedback.